### PR TITLE
Adds a much needed ion carbine crate to cargo, nerfs ion carbines

### PR DESCRIPTION
**Pull request by ghost**
Swarmers, clock cult, rogue mechs, rogue sillycons. What do these things all share? emp weakness. How many ion rifles are there on the map? 1. Cargo can replace everything in the armory except for the ion rifle, which seems...weird. Now they can sorta replace them with ion carbines. ALSO NERFS ION CARBINES SO THEY DON'T HAVE 3X3 ION RANGE.

:cl: vcordie
rscadd: Quartermasters be aware: ion carbines are now available to purchase from the nanotrasen supply depot
tweak: ion carbines now lack the 3x3 ion aoe.
/:cl:

### CODE REVIEW

**Comment by XDTM** [comment]
Tbh there IS the ion carbine in R&D which is a better version. But i agree on putting standard sec equipment in cargo.

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -417,6 +417,13 @@
 	contains = list(/obj/item/weapon/storage/box/exileimp)
 	crate_name = "exile implant crate"
 
+/datum/supply_pack/security/armory/ionrifles
+	name = "Ion Rifles Crate
```
Comment (Xhuis) [inline code/modules/cargo/packs.dm]: Missing a quote here

**Comment by jughu** [comment]
I thought ion rifles in cargi was already a thing

File: code/modules/cargo/packs.dm (hunk 1 above)
Comment (jughu) [inline code/modules/cargo/packs.dm]: Ion rifle crate* ion rifles as name is misleading since theres 1 ion rifle

**Comment by jughu** [comment]
👍on adding it to cargo
👎having 2 ion carbines in a crate silicons are already vulnarable as fuck
👍 having carbine shots nerfed
👎cheap price

**Comment by PKPenguin321** [comment]
Don't these things have fuckhueg research levels?

**Comment by vcordie** [comment]
not really. combat shotguns are the only thing worth getting from cargo because there's no other way to get combat 7. everything else can be gotten with minimum effort+minerals

**Comment by PKPenguin321** [comment]
Well okay then, :+1:

**Comment by GunHog** [comment]
Leave the ion carbine alone, it is intended as an upgrade over the ion rifle. Add the ion rifle to the cargo as a replacement if the armory is lost, fine, but not the ion carbine.

**Comment by PKPenguin321** [comment]
Isn't a carbine like a pistol version of the full rifle? It makes sense to be slightly weaker

**Comment by GunHog** [comment]
It is not a pistol, it is a carbine. A carbine is a rifle with a shorter barrel. It is not intended to be weaker, and not having AoE makes it significantly weaker, not slightly weaker.

**Comment by jughu** [comment]
@vcordie ALTOUGH do replace ion carbines with a ion rifle crate im sure this existed in the past a ion rifle crate for 4000 points would be 👌 considering it can emp doors open disable headsets and drain apcs and fucking rek silicons and you may ask why not carbine? Because carbine is a straight upgrade you can fit it in your backpack so you can ambush silicons easily with them while a ion rifle only could be put on your back and you wont easily ambush silicons with them

**Comment by PKPenguin321** [comment]
Who cares if science gets one thing berfed anyways

**Comment by PKPenguin321** [comment]
Hahaha berfed I meant nerfed my bad

The fuck does berfed mean

**Comment by Anonmare** [comment]
I'd be against nerfing the carbine so I'd rather it have an ion rifle instead. *A* ion rifle, singular, since 2.5k for 2 of them is really darn cheap. So a single rifle or a price increase would get a :+1: for me.

**Comment by jughu** [comment]
@anonmare this surely wont end up in caro techs buying carbines so they can emp open crates even easier

**Comment by Anonmare** [comment]
@jughu Oh shit yeah, then yeah it needs to be a lot more expensive since an EMP is effectively a ghetto emag. Plus it really must be the rifle version, a person can carry about 7 carbines in a regular backpack and two in each hand, but they can't carry as many with a rifle.

You have to remember that EMPs fuck up *alot* of shit. Tasers and lasers get drained in a single shot, mechs get fucked, APCs get drained completely with a single shot, silicons get knocked out/battery drained/damaged by EMPs. They also open locked containers which makes it easy than ever for cargo to become guncargo.

**Comment by PKPenguin321** [comment]
wait why are crates even affected by EMPs

**Comment by Anonmare** [comment]
@PKPenguin321 Electronic locks.

**Comment by vcordie** [comment]
I don't want them able to mass order the 3x3 dont have to aim to fuck up literally everything version, so I gave them carbines and nerfed carbines.

**Comment by jughu** [comment]
@vcordie or just make it a subtype expirimental ion rifle crate thats basically has what you asded to carbines and have some fun descriptive text to it not everything fron cargo should be 100% perfect

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -417,6 +417,13 @@
 	contains = list(/obj/item/weapon/storage/box/exileimp)
 	crate_name = "exile implant crate"
 
+/datum/supply_pack/security/armory/ionrifles
+	name = "Ion Carbines Crate"
+	cost = 2500
```
Comment (jughu) [inline code/modules/cargo/packs.dm]: I suggest 8000 points as cost
I suggest renaming it to ion rifle crate
I suggest adding 1 rifle

Heres your balance

**Comment by WJohn** [comment]
Ion rifles are comically, ridiculously powerful. It makes sense that there should be a weaker version that is more readily available.

Though two per crate may be too powerful.

Remember that there exist, at most, 4 cyborgs on station at any time. One rifle or carbine can utterly annihilate them in seconds. You can kill the AI through walls, and you can shut down a mech almost instantly (assuming it doesn't have highly upgraded parts in its build which gives it 40% EMP resistance).

**Comment by Supermichael777** [comment]
their is no limit to the number of borgs, only a production bottelneck

**Comment by kevinz000** [comment]
Uh, no, the game is good enough without you ruining balance, we do not need guncargo being able to hard counter 90% of shit tossed at them.

**Comment by PKPenguin321** [comment]
>science mains reactions to one of their toys being tweaked and put somewhere that isn't their department

good stuff keep it coming

**Comment by kevinz000** [comment]
I don't care as much about making it 1x1 as much as him giving guncargo more toys :^)

**Comment by scoopscoop** [comment]
Could make it an ion pistol if you want to keep the carbines as a RnD upgrade thing

**Comment by KorPhaeron** [comment]
Can we fix cargo having infinite resources before giving them more stuff to buy

**Comment by jughu** [comment]
@korphaeron sure!

**Comment by YakumoChen** [comment]
I would like it if cargo couldn't order an instant 'fuck all silicons' button. It's already hard enough trying to suppress the station as malf with research being able to make a shit ton if I don't shut it down, but having cargo being able to order it is even more obnoxious. 

In fact, just one ion rifle is usually enough to kill every single silicon with a decently competent player, since it's pretty much a one-hit-and-you're-fucked with borgs. Ion as a whole is ridiculously overpowered and we do not need MORE ways to easily obtain them.

**Comment by FantasticFwoosh** [comment]
Its a R&D weapon (high tech) that is more effective than a ion rifle, why are we even considering this in the first place?

Sounds like powercreep/salt for not tiding armory soon enough to get that sweet sweet **ion rifle** or getting good & crafting your own custom ion ammo (spreadshot ion) / making EMP chem grenades.

**Comment by jughu** [comment]
you have litteraly a console which can explode every silicon

**Comment by vcordie** [comment]
Dropped it to 1 ion carbine per crate.

**Comment by vcordie** [comment]
about this countering tasers and shit: stunbatons dont loose all their charge on emp, it has to be a direct hit which means if they can directly hit the sec officer, the sec officer can already fire their taser at them, and finally, all things being equal, what fucking sec officer isn't just lobbing flashbangs in the door. cargo is the PERFECT place to flashbang.

**Comment by jughu** [comment]
@vcordie still increase the price to around combat shotgun tier as its one of the most strongest power hurting weapons and can turn things like belt defib into a stun opening airlocks(basically all access aslong you have charge!) fucks silicons and drains apcs and opens crates/locker this would be basically the new crate that cargo will break open first make it a subtype ion rifle with the weakened shot and 5000 points

**Comment by GunHog** [comment]
Still against this for the same reason. Give them the full sized ion rifle, do not touch the ion carbine.

**Comment by BeeSting12** [comment]
Make it an ion rifle for about 4000 points and it would be a good idea. Also riot shotguns are not replaceable- you only get three

**Comment by vcordie** [comment]
closing, 10 thumbs down tells me all i need to know

**Pull request closed without merging**